src/components/RoleTag/RoleTag.styles.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from '@emotion/styled';
+import { Tag } from '@chakra-ui/react';
+
+export const StyledTag = styled(Tag)`
```

Comment:
Shouldn't this style be defined in `RoleList` since these margins are specific to the RoleList requirement? We can use something like StyledRoleTag.